src/main/java/io/cosmosoftware/kite/instrumentation/NetworkProfile.java

```diff
@@ -47,41 +45,46 @@ public NetworkProfile(JsonObject jsonObject) throws Exception {
     this.duplicate = jsonObject.getInt("duplicate", 0);
     this.bandwidth = jsonObject.getInt("bandwidth", 0);
     this.command = jsonObject.containsKey("command") ? jsonObject.getString("command") : this.setCommand();
-    this.nit = this.command.contains(INTERFACE_0_NAME) ? INTERFACE_0_NAME : INTERFACE_1_NAME;
-    
+    this.nit = this.command.split("dev")[1].split(" ")[1].trim();
```

Comment:
this is not safe. the command can be anything

Reply:
> we better add nit in the model as a String array

Yeah it is easier that way but who will provide nit in the model ? The client ? Us ?